tests/core/pyspec/eth2spec/test/custody_game/epoch_processing/test_process_custody_final_updates.py

```diff
@@ -77,8 +77,8 @@ def test_validator_withdrawal_suspend_after_chunk_challenge(spec, state):
     shard = 0
     offset_slots = spec.get_offset_slots(state, shard)
     shard_transition = get_sample_shard_transition(spec, state.slot, [2**15 // 3] * len(offset_slots))
-    attestation = get_valid_on_time_attestation(spec, state, index=shard, signed=True,
-                                                shard_transition=shard_transition)
+    attestation = get_valid_attestation(spec, state, index=shard, signed=True,
+                                        shard_transition=shard_transition)
```

Comment:
Don't `ShardTransition`s not exist anymore?

aah, these tests aren't actually being run... cleaning them up iterative is okay but they are still invalid in their current state and will need to be cleaned up much more extensively in the future

Reply:
yeah, i'd suggest an iterative approach here, i can keep this in mind as well but seems more sustainable to make small improvements continuously, rather than gate this on a deeper cleanup